applications/projects/SofaPhysicsAPI/src/SofaPhysicsAPI/SofaPhysicsBindings.h

```diff
@@ -0,0 +1,91 @@
+/******************************************************************************
+*                 SOFA, Simulation Open-Framework Architecture                *
+*                    (c) 2006 INRIA, USTL, UJF, CNRS, MGH                     *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU General Public License as published by the Free  *
+* Software Foundation; either version 2 of the License, or (at your option)   *
+* any later version.                                                          *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for    *
+* more details.                                                               *
+*                                                                             *
+* You should have received a copy of the GNU General Public License along     *
+* with this program. If not, see <http://www.gnu.org/licenses/>.              *
+*******************************************************************************
+* Authors: The SOFA Team and external contributors (see Authors.txt)          *
+*                                                                             *
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+#pragma once
+
+#if defined(_MSC_VER)
+#define EXPORT_API extern "C" __declspec(dllexport)
+#elif defined(UNITY_METRO)
+#define EXPORT_API __declspec(dllexport) __stdcall
+#elif defined(UNITY_WIN)
+#define EXPORT_API __declspec(dllexport)
+#elif defined(__GNUC__)
+#define EXPORT_API __attribute__((visibility("default")))
+#else
+#define EXPORT
+#pragma warning Unknown dynamic link import/export semantics.
+#endif
+
+#include <string>
```

Comment:
I dont think you need this header
(and I think without it you would be compatible with C compilers)
```suggestion
```